src/btree/bt_split.c

```diff
@@ -2184,3 +2187,25 @@ __wt_split_rewrite(WT_SESSION_IMPL *session, WT_REF *ref, WT_MULTI *multi)
 err:	__split_multi_inmem_fail(session, page, new);
 	return (ret);
 }
+
+/*
+ * __page_split_timing_stress --
+ *  Optionally add a 1000 micro seconds delay to simulate the race
```

Comment:
I will prefer not to call out the duration of the delay in the comments. That way if we change the duration in the future, we do not have to worry about updating the comment.